### PR TITLE
fix orderBy member

### PR DESCRIPTION
**Pull request by PaulGOUX27**
Fix les order by pour les members.
L'order by sur les pole fonctionne bizarrement (member>position>pole, position pouvant être vide et pole pouvant être null)

### CODE REVIEW

File: src/DataServices/Core/MemberDataService.php [hunk 1]
```diff
@@ -139,19 +138,34 @@ public function getPage(RequestParameters $requestParameters, array $queryParams
...
+                        $this->queryBuilder->orderBy("u.$orderBy", $order);
+                        break;
+                    case 'poleLabel' :
+                        $this->queryBuilder->orderBy("pole.label", $order);
+                        break;
+                    case 'positionLabel' :
+                        $this->queryBuilder->orderBy("p.label", $order);
+                        break;
+                }
             }
```
Comment (OrcaZ969) [inline src/DataServices/Core/MemberDataService.php:169]: m, u, p are the abbreviations of member username and positionLabel in the sql query.

**Review by OrcaZ969 — APPROVED** [review]

**Review by slokuge — CHANGES_REQUESTED** [review]
Need to implement the same services for member inscriptions

**Comment by PaulGOUX27** [comment]
MemberInscription use Criteria from Doctrine, and it already works well. I have fix same issue on Consultant.

**Review by slokuge — APPROVED** [review]
Good for me

**Pull request merged**